subsys/bluetooth/host/conn_internal.h

```diff
@@ -2,6 +2,8 @@
  *  @brief Internal APIs for Bluetooth connection handling.
  */
 
+#include <bluetooth/iso.h>
+
```

Comment:
We do not include headers in internal header files.

Reply:
Hmm, to avoid including it here, it has to be included by any file that includes `conn_internal.h`, which is about 26 places, almost none of which has anything to do with ISO. 

The only alternative I can think of, is to move `struct bt_conn_iso`, which uses the type from `iso.h`, to the `iso_internal.h` file instead. 

But also, as a rebuttal, we do that in rfcomm_internal.h, l2cap_internal.h, iso_internal.h, avdtp_internal.h as well as the internal headers for LE Audio.

Reply:
Why `struct bt_conn_iso` is here, wouldn't moving `struct bt_conn_iso`  into iso_internal.h reduce including in iso.h in non-iso related c files?

I am ok to take this outside of this PR as a github issue that can be fixed subsequently, i.e. to avoid including definitions in unrelated c source files.

Reply:
I don't disagree with your observations. I don't know why it was added to conn_internal.h instead of `iso_internal.h`, where it could also have been defined. We can do that in a separate PR. 

Do you think that `struct bt_conn_br` should also be moved to `br.h` then? 

@alwa-nordic @Vudentz You have any input here?